src/app-layout/notifications/index.tsx

```diff
@@ -42,22 +41,12 @@ interface NotificationWrapperProps extends NotificationsProps {
 }
 
 export const Notifications = React.forwardRef(
-  (
-    { navigationPadding, toolsPadding, sticky, isMobile, ...rest }: NotificationWrapperProps,
-    ref: React.Ref<HTMLDivElement>
-  ) => {
+  ({ sticky, isMobile, ...rest }: NotificationWrapperProps, ref: React.Ref<HTMLDivElement>) => {
     const notificationsProps: NotificationsProps = { isMobile, ...rest };
-    return (
-      <div
```

Comment:
Remove the div because seems all classes were there for VR and removed from style.css with [this PR](https://github.com/aws/awsui-components-react/pull/54/files#diff-7c7d354f562c8d16b05b7287f22c9b805308726e425b2ae371b0e54a0ba99f67).